docs/tutorial-run-solver.md

```diff
@@ -0,0 +1,176 @@
+
+```
+export PRIVATE_KEY=0x80b19a03bb57789572aabdc0e77df6c0ba556702c76c875d6598834392196610
+export ETHEREUM_NODE_URL=https://rinkeby.infura.io/v3/${INFURA_KEY}
+export NETWORK_ID=4
+
+export ORDERBOOK_FILE=/app/dex-services/target/stablex_orderbook_rinkeby.bin
+export SOLVER_TYPE=open-solver
+```
+
+Observe that the `INFURA_KEY` must also be specified beforehand for the `ETHEREUM_NODE_URL` to be valid.
+
+A successfully running and properly configured driver should appear with the following logs:
+
+```
+root@d792c990d8bd:/app/dex-services# source .env_rinkeby
+root@d792c990d8bd:/app/dex-services# cargo run --bin driver
+    Finished dev [unoptimized + debuginfo] target(s) in 9.21s
+     Running `target/debug/driver`
+2020-09-07T13:29:33.335Z INFO [driver] Starting driver with runtime options: Options {
+    log_filter: "warn,driver=info,services_core=info",
+    node_url: "https://rinkeby.infura.io/v3/c8a386aaa40349eda89325818dbc40e9",
+    network_id: 4,
+    solver_type: OpenSolver,
+    solver_internal_optimizer: Scip,
+    token_data: TokenData(
+        {},
+    ),
+    orderbook_filter: OrderbookFilter {
+        tokens: Blacklist(
+            {},
+        ),
+        users: {},
+    },
+    private_key: PrivateKey(
+        0xee095e12b02a2b82d3ae06bbc7cb2a22885922a5,
+    ),
+    auction_data_page_size: 500,
+    rpc_timeout: 10s,
+    http_timeout: 10s,
+    target_start_solve_time: 30s,
+    latest_solution_submit_time: 210s,
+    earliest_solution_submit_time: 0ns,
+    economic_viability_subsidy_factor: 10.0,
+    economic_viability_min_avg_fee_factor: 1.1,
+    default_min_avg_fee_per_order: 0,
+    default_max_gas_price: 100000000000,
+    scheduler: Evm,
+    price_source_update_interval: 300s,
+    orderbook_file: Some(
+        "/app/dex-services/target/stablex_orderbook_rinkeby.bin",
+    ),
+}
+2020-09-07T13:29:35.441Z INFO [driver] Using contract at 0xc576ea7bd102f7e476368a5e98fa455d1ea34de2
+2020-09-07T13:29:35.446Z INFO [driver] Using account Some(Offline(PrivateKey(0xee095e12b02a2b82d3ae06bbc7cb2a22885922a5), Some(4)))
+2020-09-07T13:29:35.447Z INFO [driver] Orderbook filter: OrderbookFilter { tokens: Blacklist({}), users: {} }
+2020-09-07T13:29:35.463Z INFO [services_core::price_finding] Using OpenSolver optimization price finder
+2020-09-07T13:29:38.048Z INFO [services_core::history::events] Successfully loaded 60846 events in 12437006 bytes from event registry file
+2020-09-07T13:29:38.050Z INFO [services_core::orderbook::streamed::updating_orderbook] successfully recovered orderbook from path
+2020-09-07T13:29:38.284Z INFO [services_core::orderbook::streamed::updating_orderbook] Updating event based orderbook from block 7155117 to block 7155296.
+2020-09-07T13:29:40.031Z INFO [services_core::orderbook::streamed::updating_orderbook] Received 164 events.
+2020-09-07T13:29:41.227Z INFO [services_core::orderbook::streamed::updating_orderbook] Finished applying events
+```
```

Comment:
I also got a lot of
```
2020-09-09T14:23:48.891Z WARN [services_core::price_estimation::clients::generic_client] failed to retrieve services_core::price_estimation::clients::dexag::api::DexagHttpApi prices for token ID 34 (aUSDT): failed to get price from dexag

Caused by:
    0: failed to parse JSON '"{\"error\":\"Error getting price\"}"'
    1: invalid type: string "{\"error\":\"Error getting price\"}", expected struct Price at line 1 column 37
2020-09-09T14:23:48.892Z WARN [services_core::price_estimation::clients::generic_client] failed to retrieve services_core::price_estimation::clients::dexag::api::DexagHttpApi prices for token ID 41 (aMKR): failed to get price from dexag
```
Is it because I just ran the first command you provided?

Reply:
Nah. These warning are because the price feed has no price.

Reply:
I'd add a "warning warning", since these messages make it look like something might be going wrong.

Reply:
Yes. I could also mention to use different log filter so these aren't even seen.